json-c/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+    name: json-c
+    version: 0.12
+
+source:
+    git_url: https://github.com/json-c/json-c.git
+    git_tag: json-c-0.12-20140410
```

Comment:
Can we grab a [tarball]( https://github.com/json-c/json-c/wiki )? They even have ones without docs. :smile: